mediator/export-handler.js

```diff
@@ -43,7 +43,7 @@ exports.handleExport = (req, res, next) => {
   searchDataValues(req.query)
     .then(response => exportDataValues(response.dataValues))
     .then(dataValues => {
-      res.status(200).send(dataValues)
+      res.status(200).attachment('export.json').send(dataValues)
```

Comment:
@bausmeier this works great. But can we instead have this use the code for mozdev-23 to import the data directly into the postgres db. Kinda like an export/import operation all in one.

Reply:
@nthfloor2 I'm not sure what you mean by that. This is already branched off of MOZDEV-23 and the change just makes the browser treat the returned data as an attachment to download instead of trying to display it in the browser.

Reply:
@bausmeier ah my bad, I thought this was branched off master. And since I didn't see the other changes I thought they needed to still be merged.